src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -397,7 +397,7 @@ private static State.Interface createDownStateInterface(int index) {
 	}
 
 	/** Create an element of {@link State#radios}. */
-	private static JsonObject createStateRadio() {
+	private static State.Radio createStateRadio() {
 		// @formatter:off
 		return gson.fromJson(
```

Comment:
While you're at it, can you replace the body of this function with a regular constructor (not gson)?